admin/components/Settings/index.js

```diff
@@ -149,6 +152,18 @@ const Settings = ({
               onChange={setExcludes}
             />
           </FormField>
+          <Box direction="row" justify="between" align="center" width="262px">
+            <Paragraph margin={{ vertical: "0", left: "12px", right: "20px" }}>
+              {fieldInjectionType.label}
+            </Paragraph>
+            <select onChange={setInjectionType} value={injectionType}>
```

Comment:
Why haven't you used grommet's [Select](https://v2.grommet.io/select)?  😂 

I think @orballo secretly hates grommet xD

Reply:
No problem. Great work on this btw! 👍